evap/settings.py

```diff
@@ -38,18 +38,16 @@
 RESULTS_WARNING_COUNT = 4
 RESULTS_WARNING_PERCENTAGE = 0.5
 
-# the final total grade will be calculated by the following formula (GP = GRADE_PERCENTAGE, CP = CONTRIBUTION_PERCENTAGE):
-# final_likert = CP * likert_answers_about_persons + (1-CP) * likert_answers_about_courses
-# final_grade = CP * grade_answers_about_persons + (1-CP) * grade_answers_about_courses
-# final = GP * final_grade + (1-GP) * final_likert
-GRADE_PERCENTAGE = 0.8
-CONTRIBUTION_PERCENTAGE = 0.5
+# percentages for calculating a course's total average grade
+CONTRIBUTOR_GRADE_QUESTIONS_PERCENTAGE = 0.4  # grade questions make up this percentage of each contributor's average grade
+CONTRIBUTIONS_PERCENTAGE = 0.33  # the average contribution grade makes up this percentage of the course's average grade
+COURSE_GRADE_QUESTIONS_PERCENTAGE = 0.33  # the average grade of all grade questions about the course makes up this percentage of the course's average grade
```

Comment:
1 minus others might prevent some setting misconfigurations but we probably don't have to take these into account. i don't care about the naming that much. so just say what you prefer (i think i already know ;)

Reply:
to clarify, this would also be a semantic change. you could specify 1, 1 and 10 as weights and it would just work. i would argue this even prevents misconfiguration compared to the current solution since it prevents the weight of the third group to become implicitly negative.

Reply:
i thought you would say that.